src/src_V2/modules/manager/todo/utils/constants.js

```diff
@@ -0,0 +1,40 @@
+export const TaskStatus = {
+    TODO: 'todo',
+    IN_PROGRESS: 'in_progress',
+    DONE: 'done',
+    ARCHIVED: 'archived',
+};
+
+export const TaskPriority = {
+    HIGH: 'high',
+    MEDIUM: 'medium',
+    LOW: 'low',
+};
+
+export const ReminderStatus = {
+    SET: 'set',
+    NOT_SET: 'not_set',
+    PASSED: 'passed',
+};
+
+export const TodoListTabs = {
+    TODO: 'todo',
+    ARCHIVED: 'archived',
+};
+
+export const TODO_FILTER_DEFAULT_VALUES = {
+    tab: TodoListTabs.TODO,
+    name: '',
+    statuses: '',
+    priorities: '',
+};
+
+export const TODO_FORM_INITIAL_VALES = {
```

Comment:
we also discussed the yup schemas and initail values structure tommorow i'll push it for manager module, you can review and we can have discussion about it.

Reply:
can you pls provide link?

Reply:
My Pr in pending mode i think @sasuntitan will have look tomorrow and it will be able in dev branch.

Reply:
@TikYengibaryan jan it already exists in dev branch(utils folder), please check it out

Reply:
done